Objective-C/Tests/ReplicatorTest+CustomConflict.m

```diff
@@ -247,6 +247,36 @@ - (void) testConflictResolverDeletedRemoteWins {
                                                            includeDeleted: YES error: &error].sequence);
 }
 
+- (void) testConflictResolverDeletedBothRev {
```

Comment:
I'll bet I don't have one of these.  Better get one.

Reply:
We did. You reviewed the PR.

Reply:
No!!! I mean that there is no such test for the Java code, yet!

Reply:
https://github.com/couchbaselabs/couchbase-lite-java-ee/blob/master/lib/src/shared/test/java/com/couchbase/lite/ReplicatorConflictResolutionTests.kt#L1209

Reply:
Jay's name is more concise.

Reply:
Oh!!! Now I get it!  You put that in there.  I did!  I even reviewed it!  Musta been asleep.